db/dcrpg/pgblockchain.go

```diff
@@ -44,6 +44,8 @@ var (
 	zeroHashStringBytes = []byte(chainhash.Hash{}.String())
 )
 
+const errMsgRetry = "retry"
```

Comment:
This error message looks too short to be informative and clear.

Reply:
It's not for logging, just programmatically recognizing an outcome.  This could be `errMsgRetry = errors.New("retry")` to avoid doing kind of check like `err.Error() == "some string"`, but the message itself is not for humans.

Reply:
ok. Noted. 

You could create an new error type that guarantees `const errMsgRetry = ...` will still be used ensuring that the error remains immutable.

Reply:
Can you clarify that?  Do you mean a new actual `type` or just like `const errMsgRetry = errors.New("retry")`?

Reply:
I don't think its possible to have `const errMsgRetry = errors.New("retry")` https://play.golang.org/p/-TzVytclrMy

I meant something like this.
```golang
type shortError string

func (s shortError) Error() string {
 	return string(s)
}

const errMsgRetry = shortError("retry")
```

Reply:
Gotcha... implement the `error` interface.  That's cleaner.

Reply:
I went with a type assertion.